webrecorder/browsertrix
# Upgrade Shoelace 2.0.0-beta.61 -> 2.0.0-beta.83

Upgrade Shoelace to get native HTML `<form>` features, bugfixes, and new icons. The changes are a lot to slog through, unfortunately because our current version of Shoelace is right before the [`sl-form` component was deprecated](https://github.com/shoelace-style/shoelace/blob/next/docs/resources/changelog.md#200-beta64) (but hopefully changes are straightforward enough.) Resolves #79 

### Manual testing
1. Install new dependencies: `yarn`
2. Run app: `yarn start`
3. Log in and use app, verify it functions as intended

The main refactor is with all form elements. I've manually tested the following:
- [x] Log in
- [x] Sign up
- [x] Reset password
- [x] Update password
- [x] Add member
- [x] Create crawl template
- [x] Edit crawl template
- [x] Add exclusions to running crawl
- [x] Update crawl schedule
- [x] Admin invite
